experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/proxies.spec.ts

```diff
@@ -334,7 +376,7 @@ describe("SharedTreeList", () => {
 			itWithRoot("moveRangeToStart()", schema, initialTree, (root) => {
 				const list1 = getEitherList(root, "a");
 				const list2 = getEitherList(root, "b");
-				list2.moveRangeToStart(/* sourceStart: */ 1, /* sourceEnd: */ 2, list1);
+				list2.moveToStart(1, list1);
```

Comment:
This is no longer testing the function advertised in the test name

Reply:
I changed the test titles of these ones to be less misleading.